CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## Unreleased
+
+### Improvements
+
+> [!WARNING]
+> This release enables the experimental feature `enableFileManagerSwizzling` by default. You can disable it by setting `experimental.enableFileManagerSwizzling` to `false`.
```

Comment:
`h`: The changelog needs an update to reflect that `enableFileManagerSwizzling` isn't enabled by default.

Reply:
As the option is not changed in this PR anymore, I removed it from the changelog all-together